features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -477,3 +477,181 @@ Feature: SDN compoment upgrade testing
     And the output should not contain "<%= cb.host_pod1.ip %>"
     """
 
+  # @author asood@redhat.com
+  @admin
+  @upgrade-prepare
+  @4.10 @4.9
+  @aws-ipi @gcp-ipi @vsphere-ipi @azure-ipi @baremetal-ipi @openstack-ipi
+  @aws-upi @gcp-upi @openstack-upi
+  @destructive
+  @network-ovnkubernetes
+  Scenario: Check network policy ACL logging works post upgrade -prepare
+    Given I switch to cluster admin pseudo user
+    Given the env is using "OVNKubernetes" networkType
+    When I run the :new_project client command with:
+      | project_name | rsyslog-server |
+    Then the step should succeed
+    And evaluation of `"rsyslog-server"` is stored in the :proj0 clipboard
+    Given I obtain test data file "networking/syslog/config-map.yaml"
+    Given I obtain test data file "logging/clusterlogforwarder/rsyslog/insecure/rsyslogserver_deployment.yaml"
+    Given I create the resources for the receiver with:
+      | namespace       | <%= cb.proj0 %>               |
+      | receiver_name   | rsyslogserver                 |
+      | configmap_file  | config-map.yaml               |
+      | deployment_file | rsyslogserver_deployment.yaml |
+      | pod_label       | component=rsyslogserver       |
+    And evaluation of `service("rsyslogserver").ip` is stored in the :svc_ip clipboard
+    And evaluation of `pod(0).name` is stored in the :rsyslog_server_pod clipboard
+    Then the step should succeed
+
+    Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with:
+      | {"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"policyAuditConfig": {"destination": "udp:<%= cb.svc_ip %>:514" }}}}} |
+    And I wait up to 30 seconds for the steps to pass:
+    """
+      Given the status of condition "Progressing" for network operator is :True
+    """
+    And I wait up to 300 seconds for the steps to pass:
+    """
+      Given the status of condition "Progressing" for network operator is :False
+      And the status of condition "Available" for network operator is :True
+    """
+
+    When I run the :new_project client command with:
+      | project_name | policy-upgrade6 |
```

Comment:
I suggest we can use one meaningful name instead of `policy-upgrade6`

Reply:
All the test I coded are related to network policy so I just followed the pattern of using sequence. I can change to policy_acl_upgrade1 and policy_acl_upgrade2 if that works. It is easy change as I have all names defined in beginning.

Reply:
Fixed